inventory/byo/hosts.atomic-enterprise.example

```diff
@@ -0,0 +1,79 @@
+# ssh agent.
+ansible_ssh_user=root
+
+# If ansible_ssh_user is not root, ansible_sudo must be set to true and the
+# user must be configured for passwordless sudo
+#ansible_sudo=true
+
+# deployment type valid values are origin, atomic-enterprise, openshift-enterprise
+deployment_type=atomic-enterprise
+
+# Enable cluster metrics
+#use_cluster_metrics=true
+
+# Pre-release registry URL
+#oreg_url=rcm-img-docker01.build.eng.bos.redhat.com:5001/aos3/aos-${component}:${version}
+
+# Pre-release Dev puddle repo
+#openshift_additional_repos=[{'id': 'aep-devel', 'name': 'aep-devel', 'baseurl': 'http://buildvm-devops.usersys.redhat.com/puddle/build/AtomicOpenShift/3.1/latest/RH7-RHAOS-3.1/$basearch/os', 'enabled': 1, 'gpgcheck': 0}]
+
+# htpasswd auth
+openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/htpasswd'}]
+
+# Allow all auth
+#openshift_master_identity_providers=[{'name': 'allow_all', 'login': 'true', 'challenge': 'true', 'kind': 'AllowAllPasswordIdentityProvider'}]
+
+# Configure Fluentd
+#use_fluentd=true
+
+# master cluster ha variables using pacemaker or RHEL HA
+#openshift_master_cluster_password=aep_cluster
+#openshift_master_cluster_vip=192.168.133.25
+#openshift_master_cluster_public_vip=192.168.133.25
+#openshift_master_cluster_hostname=aep-ansible.test.example.com
+#openshift_master_cluster_public_hostname=aep-ansible.test.example.com
+
+# master cluster ha variables when using a different HA solution
+# For installation the value of openshift_master_cluster_hostname must resolve
+# to the first master defined in the inventory.
+# The HA solution must be manually configured after installation and must ensure
+# that openshift-master is running on a single master host.
+#openshift_master_cluster_hostname=aep-ansible.test.example.com
+#openshift_master_cluster_public_hostname=aep-ansible.test.example.com
+#openshift_master_cluster_defer_ha=True
+
+# default subdomain to use for exposed routes
+#osm_default_subdomain=apps.test.example.com
+
+# additional cors origins
+#osm_custom_cors_origins=['foo.example.com', 'bar.example.com']
+
+# default project node selector
+#osm_default_node_selector='region=primary'
+
+# host group for masters
+[masters]
+aep3-master[1:3]-ansible.test.example.com
+
+# defining an etcd group switches from embedded etcd to external etcd
+#[etcd]
+#aep3-etcd[1:3]-ansible.test.example.com
+
+# NOTE: Currently we require that masters be part of the SDN which requires that they also be nodes
+# However, in order to ensure that your masters are not burdened with running pods you should
+# make them unschedulable by adding openshift_scheduleable=False any node that's also a master.
+[nodes]
+aep3-master[1:3]-ansible.test.example.com
+aep3-node[1:2]-ansible.test.example.com openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
```

Comment:
@devop-mmcgrath See above, do you want me to push for this to be merged as is or should I reset these hostnames back to ose3-* ? You'll also want to update the docs to reference this atomic-enterprise specific example hosts file. I've removed as many instances as 'openshift' as is safe to do without moving code around.